app/src/main/java/com/example/trashminder/presentation/auth/authUtils/Utils.kt

```diff
@@ -1,5 +1,5 @@
 package com.example.trashminder.presentation.auth.authUtils
 
  fun assertFieldsEmpty(vararg fields: String): Boolean {
-    fields.find { it.isEmpty() }?.let { return false } ?: return true
+    fields.find { it.isEmpty() || it.startsWith(" ")}?.let { return false } ?: return true
```

Comment:
I think `it.startsWith(" ")` could also be written as `it.isBlank()`. Could you check?